scripts/website.js

```diff
@@ -4,9 +4,11 @@ Error.stackTraceLimit = Infinity;
 
 const acquit = require('acquit');
 const fs = require('fs');
+const fsextra = require('fs-extra');
 const path = require('path');
 const pug = require('pug');
 const pkg = require('../package.json');
+const rimraf = require('rimraf');
```

Comment:
unless i missed something, i dont think we actually need `rimraf` in the script itself (it is still handy as a cli tool) because nodejs natively supports rm and rmdir (recursive) now